website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+---
+title: '0.72 - Beta Support for Package Exports and Symlinks'
+authors: [lunaleaps, cipolleschi]
```

Comment:
can we add @kelset and whoever from the release crew participated in this (@Titozzz ?)

Reply:
yeah I think we should have the whole crew, so me Luna Riccardo and @fortmarek